### DISCUSSION TITLE
Binary file transfer from celery worker to the server.

### DISCUSSION THREAD

**Discussion by ssid334** [nautobot/nautobot#4185] [Q&A]
Hey beautiful people,

I'm developing a job that produces binary file. It's PDF that I would like to make available via Nautobot server page.

Is there an easy way to transfer that file, or this requires extra effort? I mean I can create git repository and upload the reports there, but is there anything native provided from Nautobot server itself?

Thanks

**Comment by glennmatthews** [comment]
I'm not aware of an easy way offhand, as it may depend on the specifics of your deployment (e.g. do the worker and the server share a common filesystem?). #3352 is also highly relevant here and I see that it contains a proposed workaround that you might explore.
